packages/forms-web-app/__tests__/unit/controllers/eligibility/granted-or-refused-permission.test.js

```diff
@@ -111,5 +119,29 @@ describe('controllers/eligibility/granted-or-refused-permission', () => {
 
       expect(res.redirect).toHaveBeenCalledWith(`/${VIEW.ELIGIBILITY.DECISION_DATE}`);
     });
+
+    it(`should redirect to '/${VIEW.ELIGIBILITY.GRANTED_REFUSED_PERMISSION_OUT}' if planniong PermissionStatus is 'granted'`, async () => {
```

Comment:
There is a typo here, `planniong` should be `planning` :)

Should `planniong PermissionStatus` be `planningPermissionStatus` or `planning permission status`? It seems to be halfway between the variable name and text description :)